content/java/concepts/array-list/terms/size/size.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.size()'
+Description: 'Returns the number of elements in the ArrayList.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`size()`** method of the `ArrayList` class returns the number of elements in the list.
+
+## Syntax
+
+```pseudo
+int size = list.size()
```

Comment:
Can we include a sentence about what is returned (can mention the type which would explain the explicit `int` and link out to https://www.codecademy.com/resources/docs/java/data-types). Also, include what `list` represents.

Reply:
sure and @yangc95 can you let me know why my old pr are not getting merged even after changing everything that I was asked to change?

Reply:
> can you let me know why my old pr are not getting merged even after changing everything that I was asked to change?

Thanks for the changes 🙂 Maintainers are working to get every submitted PR up to standards and merged in a timely manner so ty for your patience!